djconnectwise/api.py

```diff
@@ -331,7 +333,10 @@ def _fetch_resource(endpoint_url, params=None, should_page=False,
                 raise ConnectWiseAPIError('{}'.format(e))
 
             if 200 <= response.status_code < 300:
-                return response.json()
+                try:
+                    return response.json()
+                except JSONDecodeError as error:
+                    logger.error('An error occurred during returning response as dict: {}'.format(error))
```

Comment:
We also need to return an iterable here. Otherwise we get an error like this:

```
    sync_runner(obj_name, synchronizer)                                                           
  File "/home/cameron/Dev/Kanban/django-topleft/kanban/kanban/connectwise/tasks.py", line 452, in 
sync_runner                                                                                       
    synchronizer.sync()                                                                           
  File "/home/cameron/Dev/Kanban/env3/lib/python3.8/site-packages/django_connectwise-0.3.198-py3.8
.egg/djconnectwise/sync.py", line 61, in wrapper                                                  
    f(*args, **kwargs)                                                                            
  File "/home/cameron/Dev/Kanban/env3/lib/python3.8/site-packages/django_connectwise-0.3.198-py3.8
.egg/djconnectwise/sync.py", line 371, in sync                                                    
    results = self.get(results, )                                                                 
  File "/home/cameron/Dev/Kanban/env3/lib/python3.8/site-packages/django_connectwise-0.3.198-py3.8
.egg/djconnectwise/sync.py", line 171, in get                                                     
    return self.fetch_records(results, conditions)                                                
  File "/home/cameron/Dev/Kanban/env3/lib/python3.8/site-packages/django_connectwise-0.3.198-py3.8
.egg/djconnectwise/sync.py", line 192, in fetch_records                                           
    self.persist_page(page_records, results)                                                      
  File "/home/cameron/Dev/Kanban/env3/lib/python3.8/site-packages/django_connectwise-0.3.198-py3.8
.egg/djconnectwise/sync.py", line 202, in persist_page                                            
    for record in records:                                                                        
TypeError: 'NoneType' object is not iterable                                                      
```